manager/assets/modext/util/utilities.js

```diff
@@ -421,28 +421,8 @@ Ext.applyIf(Ext.form.Field,{
 /* allow copying to clipboard */
 MODx.util.Clipboard = function() {
     return {
-        escape: function(text){
-            text = encodeURIComponent(text);
-            return text.replace(/%0A/g, "%0D%0A");
-        }
-
-        ,copy: function(text){
-            if (Ext.isIE) {
-                window.clipboardData.setData("Text", text);
-            } else {
-                var flashcopier = 'flashcopier';
-                if (!document.getElementById(flashcopier)) {
-                    var divholder = document.createElement('div');
-                    divholder.id = flashcopier;
-                    document.body.appendChild(divholder);
-                }
-                document.getElementById(flashcopier).innerHTML = '';
-                var divinfo = '<embed src="' + MODx.config.manager_url
-                    + 'assets/modext/_clipboard.swf" FlashVars="clipboard='
-                    + MODx.util.Clipboard.escape(text)
-                    + '" width="0" height="0" type="application/x-shockwave-flash"></embed>';
-                document.getElementById(flashcopier).innerHTML = divinfo;
-            }
+        copy: function(text){
+            console.log('MODx.util.Clipboard was deactivated!');
```

Comment:
I suggest removing this component completely. If there are good reasons to keep it we should log an error instead.

Reply:
A github code search for MODx.util.Clipboard shows 459 results, most in the utilities.js file or in some cache files. So I think that the component is no longer in use.